src/pages/fr/guides/configuring-astro.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Configurer Astro
+---
+
+Personnalisez le fonctionnement d'Astro en ajoutant un fichier `astro.config.mjs` dans votre projet. Il s'agit d'un fichier commun aux projets Astro, et tous les modèles et thèmes d'exemple officiels en contiennent un par défaut.
+
+📚 Consultez la [référence d'API d'Astro](/fr/reference/configuration-reference/) pour une vue d'ensemble des options supportées.
+
+## Le Fichier de Configuration d'Astro
+
+Un fichier de configuration Astro valide exporte sa configuration en utilisant l'exportation `default`, et en utilisant l'aide recommandée `defineConfig` :
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config'
+
+export default defineConfig({
+  // vos options de configuration ici...
+  // https://docs.astro.build/fr/reference/configuration-reference/
+})
+```
+
+Utiliser `defineConfig()` est recommandé pour automatiser les types dans votre éditeur de code, mais c'est aussi optionnel. Un fichier de configuration absolument minimal, valide, pourrait ressembler à ça :
+
+```js
+// Exemple : Fichier de configuration minimal, vide
+export default {}
+```
+
+## Types de Fichier de Configuration Supportés
+
+Astro supporte plusieurs formats de fichiers pour son fichier de configuration JavaScript : `astro.config.js`, `astro.config.mjs`, `astro.config.cjs` et `astro.config.ts`. 
+
+Le chargement du fichier de configuration TypeScript est géré par [`tsm`](https://github.com/lukeed/tsm) et respecte les options tsconfig de votre projet.
+
+## Résolution du Fichier de Configuration
+
+Astro essaiera automatiquement de résoudre un fichier de configuration nommé `astro.config.mjs` à la racine du projet. Si aucun fichier de configuration n'est trouvé à la racine de votre projet, les options par défaut d'Astro seront utilisées.
+
+```bash
+# Exemple : Lit votre configuration à partir de ./astro.config.mjs
+astro build
+```
+
+Vous pouvez explicitement définir un fichier de configuration à utiliser avec l'option `--config` du CLI. Cette option est toujours relatif au répertoire de travail actuel dans lequel vous avez exécuté la commande `astro`.
+
+```bash
+# Exemple : Lit votre configuration à partir de ce fichier
+astro build --config my-config-file.js
+```
+
+## Autocomplétion de Configuration
+
+Astro recommande d'utiliser l'aide `defineConfig()` dans votre fichier de configuration. `defineConfig()` fournit une auto-complétion automatique dans votre IDE. Les éditeurs comme VSCode sont capables de lire les définitions TypeScript d'Astro et de fournir des indications de type jsdoc automatiques, même si votre fichier de configuration n'est pas écrit en TypeScript.
```

Comment:
"jsdoc" -> "JSDoc" for consistency? I would expect "jsdoc" when it is a file extension, but "JSDoc" to refer to the markup language.